docs/api/faucet/run-faucet.schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "description": "POST request that runs the faucet",
+  "description": "POST request that adds tokens to a specified testnet address",
```

Comment:
nit: could say that this will _initiate a transfer_ of tokens to the account - to convey that there's a waiting period of a few minutes for the transaction to be mined